### DISCUSSION TITLE
Adding new billing_code_type valid values for in-network file

### DISCUSSION THREAD

**Discussion by iGokul-G** [CMSgov/price-transparency-guide#111] [Q&A] [answered]
Could you please include ICD10 Diagnosis and ICD10 Procedure code as  valid values.  Below are few potential services 

ex .  1. Inpatient service  related to Covid  may have one  of criteria as ICD10 Diagnosis code U071
        2. PTCA service could  be represented  IC10procedure codes like 02C13ZZ,02C33ZZ,02703DZ ,02723DZ etc.

**Comment by clovis517** [comment, marked as answer]
Answered here : [The billing codes are not restricted to values found in the github documentation](https://github.com/CMSgov/price-transparency-guide/discussions/9#discussioncomment-778852)

That said, I think a standard for concatenating types and codes would be helpful.  e.g. for bilateral:
   "billing_code_type": "CPT+MODIFIER"
   "billing_code": "27447+50"

Other combinations needed : 
   CPT+ICD for diagnostic specific pricing
   CPT+REVCODE

Separate related question : Since HCPCS Level 1 = CPT are we compliant if we report HCPCS for both level 1 and level 2 and never use the CPT type?

**Reply by reevesj2** [reply]
What about things that we pay based on combination of DRG/REVCODE or Type of Bill/REVCODE.  I understood that we were not limited in billing code types, but does the file support reporting of two disparate types of bill codes together?

**Reply by reevesj2** [reply]
Similar question to your example and question about reporting HCPCS and never using CPT.  For our system all CPT and HCPCS are stored together. There is no identifier to indicate that a specific code is a HCPC or CPT code.   Currently we could report all as CPT or all as HCPC.  Anyone else have this issue and thoughts on reporting?  Can we report the Billing Code Type as CPT/HCPC?

**Reply by tdfow** [reply]
#9 
#12
